br-ext/package/xen_ext/Config.in

```diff
@@ -0,0 +1,51 @@
+config BR2_PACKAGE_XEN_EXT
```

Comment:
Please mention in the commit description where this file comes from (`<buildroot>/package/xen/Config.in` I guess), and which revision. It could become handy if we need to "rebase" in the future.